src/docs/product/data-management-settings/event-grouping/index.mdx

```diff
@@ -4,14 +4,15 @@ sidebar_order: 0
 redirect_from:
   - /data-management/rollups/
   - /learn/rollups/
+  - /data-management/event-grouping/
```

Comment:
Shouldn't this redirect from the platform pages, or is that what it does?

Reply:
This is the location in which it used to reside.